contracts/utils/MutualConsent.sol

```diff
@@ -81,6 +82,9 @@ abstract contract MutualConsent {
     /* ============ Internal Functions ============ */
 
     function _mutualConsent(address _signerOne, address _signerTwo) internal returns (bool) {
+        if (msg.value != 0) {
```

Comment:
If u have it here then u dont need to add check in individual functions too right?